### PR TITLE
Added dots background

### PR DESCRIPTION
**Pull request by Manasvi1606**

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -1,97 +1,12 @@
-import Image from 'next/image';
-import { GithubDoticon } from 'doticons/32';
-import styles from './page.module.scss';
+import DotsBackground from '../components/DotsBackground';
 
-export default function Home() {
+function YourPage() {
```
Comment (eduardconstantin) [inline app/page.tsx:3]: Lets keep this as home
```suggestion
export default function Home() {
```

File: app/page.tsx [hunk 2]
```diff
@@ -1,97 +1,12 @@
...
-      </div>
-    </main>
+    <div>
+      <DotsBackground />
+      {/* Your page content */}
+    </div>
   );
 }
+
+export default YourPage;
```
Comment (eduardconstantin) [inline app/page.tsx:12]: this can be removed
```suggestion
```

File: components/DotsBackground.js
Comment (eduardconstantin) [inline components/DotsBackground.js:1]: you can remove the comments

File: components/DotsBackground.module.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  }
+  
+  .svg {
+    width: 100%;
+    height: 100%;
+  }
+  
+  .dot {
+    fill: rgb(255, 255, 255);
+    animation: dotAnimation 5s infinite alternate ease-in-out;
```
Comment (eduardconstantin) [inline components/DotsBackground.module.css:19]: the animation should play randomly for each dot
if you think is too much, you can remove this line

**Review by eduardconstantin — CHANGES_REQUESTED** [review]

**Comment by eduardconstantin** [comment]
Please fix the conflicts too

**Comment by Manasvi1606** [comment]
Looks good, will make the necessary changes and fix the conflicts.

File: components/DotsBackground.module.css (hunk 3 above)
Comment (Manasvi1606) [reply components/DotsBackground.module.css:19]: how frequently do you want the animation to randomly play the animation.
currently I have added the code such that at random times some circles are of opacity 1 and some of 0.4 then in a few seconds the animation changes, Will that work or should I remove the animation?

Comment (eduardconstantin) [reply components/DotsBackground.module.css:19]: this should be fine for now

Comment (Manasvi1606) [reply components/DotsBackground.module.css:19]: okay great

**Comment by Manasvi1606** [comment]
I have made the necessary changes, please review!

File: app/page.tsx
Comment (eduardconstantin) [inline app/page.tsx:1]: Can you add the background?

**Review by eduardconstantin — CHANGES_REQUESTED** [review]

File: app/page.tsx
Comment (Manasvi1606) [reply app/page.tsx:1]: I am not sure I understand, do you mean to say can I close the pull request?

Comment (Manasvi1606) [reply app/page.tsx:1]: okay, sorry I understand. will do!

Comment (eduardconstantin) [reply app/page.tsx:1]: no, the component you made is not added to the page

Comment (Manasvi1606) [reply app/page.tsx:1]: it is added now

**Review by eduardconstantin — APPROVED** [review]
Thank you for implementing the background!

**Comment by Manasvi1606** [comment]
Thank you for merging the PR! It was a fun task

**Pull request merged**